doc/python_api_reference_vDev.md

```diff
@@ -7864,6 +7865,58 @@ class TableauSimulator:
     """
 ```
 
+<a name="stim.TableauSimulator.__init__"></a>
+```python
+# stim.TableauSimulator.__init__
+
+# (in class stim.TableauSimulator)
+def __init__(
+    self,
+    *,
+    seed: Optional[int] = None,
+) -> None:
+    """Initializes a stim.TableauSimulator.
+
+    Args:
+        seed: PARTIALLY determines simulation results by deterministically seeding
+            the random number generator.
+
+            Must be None or an integer in range(2**64).
+
+            Defaults to None. When None, the prng is seeded from system entropy.
+
+            When set to an integer, making the exact same series calls on the exact
+            same machine with the exact same version of Stim will produce the exact
+            same simulation results.
+
+            CAUTION: simulation results *WILL NOT* be consistent between versions of
+            Stim. This restriction is present to make it possible to have future
+            optimizations to the random sampling, and is enforced by introducing
+            intentional differences in the seeding strategy from version to version.
+
+            CAUTION: simulation results *MAY NOT* be consistent across machines that
+            differ in the width of supported SIMD instructions. For example, using
+            the same seed on a machine that supports AVX instructions and one that
+            only supports SSE instructions may produce different simulation results.
+
+            CAUTION: simulation results *MAY NOT* be consistent if you vary how many
+            shots are taken. For example, taking 10 shots and then 90 shots will
```

Comment:
Nit: this example could be specific to the tableau simulator, which has no shots. For example could say

`[...] *MAY NOT* be consistent if you vary how the circuit is executed. For example, reordering whether a reset on one qubit happens before or after a reset on another qubit can result in different measurement results being observed starting from the same seed.`

Reply:
Done.